app/helpers/application_helper.rb

```diff
@@ -9,30 +9,6 @@ def env_style
     "background-image: url(#{image_path("#{unstable_env}.png")});"
   end
 
-  def ident_for(contact)
```

Comment:
I moved this method to `app/models/contact.rb` and changed the name to `ident_human_description`